contents/handbook/people/compensation.mdx

```diff
@@ -69,11 +72,23 @@ As we do these much more frequently than regular companies, team members should
 
 ### Pay review process
 
+Pay at PostHog is reviewed much more often at most other companies - at least 2 to 3 times a year.
+
+To make sure everyone has an equal chance at getting a payrise we do not factor in how much someone complains about their pay. When increasing pay we only look at our calculator and performance.
+
+If someone is a high performer they will be paid well.
+
+<!-- Our goal is to increase their pay as frequently as we can. This an example of being inclusive - rather than rewarding the loudest voices. -->
```

Comment:
I think we also shouldn't include this line because this seems like what a company that doesn't respect pay transparency would say? As an employee I would feel distrust from leadership if at any point I had questions or concerns about compensation it could be misconstrued as having a "loud voice". Don't we want to err on the side of giving people the benefit of doubt here? This should be the baseline of the bar we have for people who work at posthog and I don't think we should spell this out

Reply:
I've changed the language in this section a little bit, as I think the inclusivity point is important to keep in (above this section), but I agree the 'loudest voices' term is not the best way to communicate this.